assets/src/edit-story/components/panels/preset/karma/stylePresets.karma.js

```diff
@@ -112,15 +106,13 @@ describe('Panel: Style Presets', () => {
     });
 
     it('should allow adding new text style from a text element', async () => {
-      const addButton = fixture.screen.getByRole('button', {
-        name: ADD_BUTTON,
-      });
-      await addButton.click();
-      const applyPresetButton = fixture.screen.queryByRole('button', {
-        name: APPLY_BUTTON,
-      });
+      await fixture.events.click(
+        fixture.editor.inspector.designPanel.textStylePreset.add
+      );
+      const applyButton = await fixture.editor.inspector.designPanel
```

Comment:
What are we `await`'ing for? There are a few of these, where you're awaiting something non-promised.